modules/data-entry/src/main/frontend/src/questionnaire/MultipleChoice.jsx

```diff
@@ -416,7 +416,7 @@ function MultipleChoice(props) {
           {generateDefaultOptions(options, selection, disabled, isRadio, selectNonGhostOption, removeOption)}
           {/* Ghost radio for the text input */}
           {
-          (input || textbox) && <ListItem className={classes.selectionChild + " " + classes.ghostListItem}>
+          (ghostInput && <ListItem className={classes.selectionChild + " " + classes.ghostListItem}>
```

Comment:
There's one more typo...
```suggestion
          ghostInput && <ListItem className={classes.selectionChild + " " + classes.ghostListItem}>
```